src/reader.cpp

```diff
@@ -508,6 +508,7 @@ bool Reader::searchSuggestionsSmart(const string& prefix,
       suggestion.push_back(current->getTitle());
       suggestion.push_back(current->getPath());
       suggestion.push_back(kiwix::normalize(current->getTitle()));
+      suggestion.push_back(current.get_snippet());
```

Comment:
Maybe it is time to introduce a class/struct for the suggestion result instead of using a vector of strings.